test/library/draft/DistributedMap/v2/DistributedMap.chpl

```diff
@@ -32,7 +32,7 @@ module DistributedMap {
     type keyType;
     type valType;
     pragma "no doc"
-    var m: shared distributedMapImpl(keyType, valType)?;
+    var m: shared distributedMapImpl(keyType, valType, ?)?;
```

Comment:
Is the thing that requires the `?` here the fact that the `targetLocales` field is un-typed?  If it were (able to be) constrained to be a 1D DR array of locales, would the need for the `?` go away?  (I'm guessing that's the  only type we'd intend it to have, tagging @jeremiah-corrado to check).

Reply:
No `targetLocales` is declared as `const targetLocales = Locales` so its type is inferred, not generic.

The reason for the `?` here is that later the `class distributedMapImpl` has the generic field `type funcType;`.

Reply:
Oh good, I like that explanation much better—thanks!  (now I'm ready for the "Hey, you should consider moving all of your `type`/`param` fields to the top of the class so that people can see how generic your class is" warning :) ).